public/entity2/concreteentitylist.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CONCRETEENTITYLIST_H
+#define CONCRETEENTITYLIST_H
+
+#include "entityidentity.h"
+
+class CConcreteEntityList
+{
+	struct CList
+	{
+		CEntityIdentity* m_pHead;
+		CEntityIdentity* m_pTail;
+		uint64_t unknown;
+	};
+public:
+	CEntityIdentities *m_pIdentityChunks[MAX_ENTITY_LISTS];
+	uint8_t unknown[16];
```

Comment:
Is this padding correct? I do have ``m_pFirstActiveEntity`` going right after in the place of this padding and 8 byte padding after it.

Reply:
Yeah that wasn't right, it's updated now